documentation/modules/configuring/con-partition-reassignment.adoc

```diff
@@ -16,11 +16,20 @@ Cruise Control is recommended for this type of operation as it xref:cruise-contr
 Scaling topic replication factor up and down:: Increase or decrease the replication factor of your Kafka topics. To do this, you must create a partition reassignment plan that identifies the existing replication assignment across partitions and an updated assignment with the replication factor changes.
 
 Changing the preferred leader:: Change the preferred leader of a Kafka partition. 
-In Kafka, the preferred leader is the replica that serves as the leader for a partition.
-Changing the preferred leader can be useful if the current preferred leader is unavailable or if you want to redistribute load across the brokers in the cluster.
+In Kafka, the partition leader is the replica that serves as the leader for a partition. 
+However, the partition leader is not always the preferred leader; for example, if the preferred leader is out of sync or if the broker goes offline, another replica may temporarily become the leader.
```

Comment:
I think we can remove this line, as the same concept is expressed in the following lines.